README.md

```diff
@@ -119,12 +121,14 @@ function App() {
         addsPunctuation: false,
         contextualStrings: ["Carlsen", "Nepomniachtchi", "Praggnanandhaa"],
       });
-    });
+    } catch (error) {
+      console.error("Error starting speech recognition:", error);
+    }
```

Comment:
This hopefully shouldn't be required as the native iOS and Android implementations for `.start()` handle their own errors and emit those errors to the `"error"` event. You can listen to that with `useSpeechRecognitionEvent("error", ...)`. 

Likewise, with the `requestPermissionsAsync` function, it would only throw if the project was using bare React Native without `expo-modules-core` (which is present in every Expo project).

Let me know though if you did catch any uncaught errors as I don't intended for these functions to ever throw.